google/api_core/gapic_v1/method.py

```diff
@@ -52,65 +52,26 @@ def _apply_decorators(func, decorators):
     ``decorators`` may contain items that are ``None`` or ``False`` which will
     be ignored.
     """
-    decorators = filter(_is_not_none_or_false, reversed(decorators))
+    filtered_decorators = filter(_is_not_none_or_false, reversed(decorators))
 
-    for decorator in decorators:
+    for decorator in filtered_decorators:
         func = decorator(func)
 
     return func
 
 
-def _determine_timeout(default_timeout, specified_timeout, retry):
```

Comment:
Correct me here - this is removed because timeout is being passed as an argument to _blocking_poll ?

Reply:
This is because ExponentialTimeout and ConstantTimeout are both deprecated and should not be used. This method was essentially creating/selecting one of the two deprecated timeout implementations, since both of the timeouts are not needed and the method is prefixed with `_` (not supposed to be used publicly) i just removed it.